.github/workflows/test_libs.yml

```diff
@@ -64,36 +64,30 @@ jobs:
         os: ['ubuntu-22.04', 'macos-12', 'windows-2022']
         lib: ${{ fromJson(needs.set_libs_matrix.outputs.libs) }}
     runs-on: ${{ matrix.os }}
-    env:
-      CONAN_REVISIONS_ENABLED: 1
     steps:
     - uses: actions/checkout@v3
     - name: Set up Python 3.10
       uses: actions/setup-python@v4
       with:
         python-version: '3.10'
-    - name: install conan
-      run: pip install conan~=1.53
+    - name: Install conan
+      uses: turtlebrowser/get-conan@v1.2
+      with:
+        version: '2.0.2'
```

Comment:
Do we want to specify a fixed version here or something that is automatically updated?